src/sass/_fonts-general-styles.scss

```diff
@@ -0,0 +1,197 @@
+@use "_variables.scss" as *;
+
+html {
+    background-color: $tan;
+    font-family: $body-font;
+    font-size: 62.5%;
+    letter-spacing: 0.05rem;
+}
+
+ul,
+li {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+
+section { width: 100%; }
+
+.wrapper {
+    width: 75%;
+    max-width: 1100px;
+    margin: 0 auto;
+}
+
+//header fonts
+h1,
+header p,
+h2,
+h3,
+h4 {
+    margin: 0;
+    font-family: $header-font;
+    text-align: center;
+}
+
```

Comment:
It _pains_ me to say this becuase i love the look of your whole site, but i think for accessibility purposes the beige background/orange font might not be contrasted enough? I had trouble on my last project using orange fonts.. they seem tricky to get good contrast with.. but perhaps you could play with making the beige lighter/dark OR the orange to see if its slightly more contrasted. 
[https://contrast-ratio.com/#%23E75329-on-%23CABBA3](url)

Reply:
😭😭😭😭